ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Basics.hs

```diff
@@ -159,22 +165,28 @@ class ( -- Requirements on the ledger state itself
   applyChainTickLedgerResult ::
        LedgerCfg l
     -> SlotNo
-    -> l
-    -> LedgerResult l (Ticked l)
+    -> l Canonical
+    -> LedgerResult l (Ticked1 l Canonical)
 
 -- | 'lrResult' after 'applyChainTickLedgerResult'
-applyChainTick :: IsLedger l => LedgerCfg l -> SlotNo -> l -> Ticked l
+applyChainTick :: IsLedger l => LedgerCfg l -> SlotNo -> l Canonical -> Ticked1 l Canonical
 applyChainTick = lrResult ..: applyChainTickLedgerResult
 
 {-------------------------------------------------------------------------------
   Link block to its ledger
 -------------------------------------------------------------------------------}
 
 -- | Ledger state associated with a block
-data family LedgerState blk :: Type
+type LedgerState :: Type -> Type -> Type
```

Comment:
I don't know if we do it later in the `feature/utxo-hd-branch`, but we should explain what the extra parameter represents, why do we call it `mk`, what is it for, etc.